### PR TITLE
Ecal Transparency corrections : use either a tag or a file for TPG

### PR DESCRIPTION
**Pull request by depasse**
When building Ecal TPG parameters, allows to use either a file or a tag for the Transparency corrections (before, only files were allowed)

### CODE REVIEW

**Comment by thomreis** [comment]
please test

**Comment by lpernie** [comment]
+1

**Comment by thomreis** [comment]
+1

File: CalibCalorimetry/EcalTPGTools/plugins/EcalTPGParamBuilder.cc [hunk 1]
```diff
@@ -451,6 +418,69 @@ void EcalTPGParamBuilder::analyze(const edm::Event& evt, const edm::EventSetup&
...
+      
+	xtalcounter++;
+	transparency.getline(buf,sizeof(buf),'\n');
+      }//loop transparency
+      transparency.close();
+      ss << "INFO: DONE transparency correction files " << xtalcounter << "\n";
+      edm::LogInfo("TopInfo") << ss.str();
+      ss.str("");
+    }   // file
+    else {       // Jean 2018
```
Comment (fabiocos) [inline CalibCalorimetry/EcalTPGTools/plugins/EcalTPGParamBuilder.cc]: @depasse as far as I understand, this option is so far dummy, and just activated in the test configurationa dded to this PR

**Comment by fabiocos** [comment]
+1

File: CalibCalorimetry/EcalTPGTools/plugins/EcalTPGParamBuilder.cc (hunk 1 above)
Comment (depasse) [reply CalibCalorimetry/EcalTPGTools/plugins/EcalTPGParamBuilder.cc]: This is a new feature very convenient to be able to upload payloads
from sqlite files without the need to convert them to xml or txt ones.
   It is neither dummy nor for test only.

Comment (fabiocos) [reply CalibCalorimetry/EcalTPGTools/plugins/EcalTPGParamBuilder.cc]: @depasse I mean that so far there is no configuration activating it in production. Do you confirm?

Comment (depasse) [reply CalibCalorimetry/EcalTPGTools/plugins/EcalTPGParamBuilder.cc]: Yes, it is not triggered in production.  It is in EcalTPG"Tools" and will be used in the future ponctually.  It is important to keep the traceability of this.

**Pull request merged**